### PR TITLE
183 buttons

### LINKED ISSUE #162: Fix: hint/text length
### Content
When entering a really long hint with no spaces, the device table extends beyond normal width due to component.

With long hints/texts, the positioning of component/status gets mixed up

### LINKED ISSUE #183: fix table button styling
### Contents
The buttons in the front-end tables take up a lot of space, create custom md styling.

- [ ] table button
- [ ] dutch in app

### PR DESCRIPTION
**Pull request by IssaHanou**
closes #183, closes #162

- front-end is shown as 'operator scherm' with only `gameState` as status
- front-end button's pressed status is not resend from back-end (causing flickering)
- adds button for showing statuses of components
- improves button stylings in tables and disables when necessary
- hopefully fixed colors of checkmarks on all devices -> cannot test yet
- [x]  fixes placing of components/statuses with long statuses
- [x] reorganize home

home:
afbeelding

### CODE REVIEW

**Comment by GwennanS** [comment]
front-end components are not getting parsed correctly on my machine:
ERRO[2020-01-29 11:36:39] status type boolean expected but map found as type with status map[id:pre-game status:false] for component pre-game

File: front-end/src/app/components/device/device.component.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { Component, OnInit, ViewChild } from "@angular/core";
-import { Device } from "./device";
+import {Comp, Device} from "./device";
```
Comment (GwennanS) [inline front-end/src/app/components/device/device.component.ts]: prettier no like

**Comment by mrlomar** [comment]
Maybe add text to the collapsed status button: something like `click for status` or `expand`

**Comment by mrlomar** [comment]
Please run prettier :)

**Pull request merged**